go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/Azure/azure-storage-azcopy
 
 require (
 	github.com/Azure/azure-pipeline-go v0.2.1
-	github.com/Azure/azure-storage-blob-go v0.7.0
+	github.com/Azure/azure-storage-blob-go v0.8.1-0.20191213204130-762620a866ba
```

Comment:
What's the plan for getting back onto a released version?  If that plan requires us to remember to do something later, do we have a backlog entry to serve as that reminder?

Reply:
It will require us to make a release of the blob SDK before azcopy, and to fix up the go.mod for it.

I'll create the backlog item after I address other comments on this PR.